docs/index.rst

```diff
@@ -0,0 +1,51 @@
+.. include:: ../README.rst
+
+Table of Contents
+=================
+
+.. toctree::
+    :maxdepth: 4
+    :hidden:
+
+    self
+
+.. toctree::
+    :caption: Examples
+
+    examples
+
+.. toctree::
+    :caption: API Reference
+    :maxdepth: 3
+
+    api
+
+.. toctree::
+    :caption: Tutorials
+
+    https://github.com/adafruit/Adafruit_Learning_System_Guides/tree/master/DIY_Thermal_Light_Painting
+
+.. toctree::
+    :caption: Related Products
+
+    https://www.adafruit.com/product/1747
+    https://www.adafruit.com/product/1748
```

Comment:
Thanks for adding these! Please change them to match the links below with a text name. I'm not sure they work otherwise.